t/integration/tasks.py

```diff
@@ -1,6 +1,6 @@
 from time import sleep
 
-from celery import Task, chain, chord, group, shared_task
+from celery import Signature, Task, chain, chord, group, shared_task
```

Comment:
Please run isort on this file.

Reply:
`isort` seems fine with this file, however the import in `t/integration/test_canvas.py` needed to be moved. I'll amend and push in a moment.